index.html

```diff
@@ -8,81 +8,90 @@
+				        	<a class="nav-link" href="#">פזומט</a>
+				        </li>
+				        <li class="nav-item active col d-flex justify-content-center align-items-center">
+				        	<a class="nav-link" href="#">חברות הקבוצה</a>
+				        </li>
+				        <li class="nav-item active col d-flex justify-content-center align-items-center">
+				        	<a class="nav-link" href="#">אחריות תאגידית</a>
+				        </li>
+				        <li class="nav-item active col d-flex justify-content-center align-items-center">
+				        	<a class="nav-link" href="#">אודות</a>
+				        </li>
+		    		</ul>
+	    		</nav>
+    		</div>
+    	</div>
+    	<!-- HEADER 2 -->
 
-				<div class="low-sec-nav container hidden-md-down">	
-		    		<nav class="nav-1 navbar navbar-nav navbar navbar-expand-lg">
-			    		<ul class="d-flex flex-row">
-			    			<li class="nav-item active col d-flex justify-content-center align-items-center">
-					        	<a class="nav-link" href="#">צרו קשר</a>
-					        </li>
-					        <li class="nav-item active col d-flex justify-content-center align-items-center">
-					        	<a class="nav-link" href="#">קריירה</a>
-					        </li>
-					        <li class="nav-item active col d-flex justify-content-center align-items-center">
-					        	<a class="nav-link" href="#">קשרי משקיעים</a>
-					        </li>
-					        <li class="nav-item active col d-flex justify-content-center align-items-center">
-					        	<a class="nav-link" href="#">התחנות שלנו</a>
-					        </li>
-					        <li class="nav-item active col d-flex justify-content-center align-items-center">
-					        	<a class="nav-link" href="#">פזומט</a>
-					        </li>
-					        <li class="nav-item active col d-flex justify-content-center align-items-center">
-					        	<a class="nav-link" href="#">חברות הקבוצה</a>
-					        </li>
-					        <li class="nav-item active col d-flex justify-content-center align-items-center">
-					        	<a class="nav-link" href="#">אחריות תאגידית
-					        	</a>
-					        </li>
-					        <li class="nav-item active col d-flex justify-content-center align-items-center">
-					        	<a class="nav-link" href="#">אודות
-					        	</a>
-					        </li>
-			    		</ul>
-		    		</nav>
+	    <!-- reg and personal erea -->
+		<div class="nav-2 container-fluid d-flex align-items-center justify-content-between">
+    		<div class="container d-flex justify-content-lg-between">
+    			<div class="l-side-gray-nav d-flex justify-content-lg-between col">
+    				<div class="nav-icon-round-mobile d-lg-none d-flex align-items-center">
+	    				<button class="d-flex align-items-center flex-column">
+				    			<img src="img/icon-nav-round-mobile.png">
+				    			<p>תפריט</p>
+				    	</button>
+			    	</div>
+	    			<div class="sub-nav-buttons d-flex justify-content-start">
+		    			<div class="left-buttons d-flex">
```

Comment:
You have a redundant container here. You can merge `sub-nav-buttons` and `left-buttons` to one element.